File: GTBT005.py
# calculate sin taylor series by using for loop in python
from math import*
t=int(input())
for i in range(t):
    x=float(input())
    for k in range(0,10,1):
        y=((-1)**k)*(x**(1+2*k))/factorial(1+2*k)
    print(y)

File: GTBT005.cpp
#include <iostream>
#include <math.h>
using namespace std;

const int m=19;

int factorial(int n) 
{
    if (n==0){ return 1;}
    return n*factorial(n-1);
}

int main() 
{
    float x;
    cin >> x;
    long long sum=0;
    int k=1;
    for (int i=1;i<=m;i+=2) 
    {
        sum+=(k*(powf(x,i)/factorial(i)));
        k=k*(-1);
    }
    cout<<sum<<endl;
    return 0;
}